src/components/ContentExplorer/ItemList.js

```diff
@@ -187,9 +190,9 @@ const ItemList = ({
                                             : intl.formatMessage(messages.modified)
                                     }
                                     dataKey={isRecents ? FIELD_INTERACTED_AT : FIELD_MODIFIED_AT}
-                                    cellRenderer={dateCell}
+                                    cellRenderer={modifiedCell}
                                     headerRenderer={headerCellRenderer}
-                                    width={135}
+                                    width={300}
```

Comment:
May have to see (test around a bit) if the flexing option below needs to change, since its hard to box names in a fixed width. Was easier with just date. So if the explorer size is wide enough, maybe this column should be allowed to expand.

Reply:
Should I add flexGrow={1} attribute to the Modified column? It makes the Filename and Modified columns equal width and the flexShrink value will remove the column if the width is too small. It looked good to me while resizing the Content Explorer container.

Reply:
added flexGrow={1}